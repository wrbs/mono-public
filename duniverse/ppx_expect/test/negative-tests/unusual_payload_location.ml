let%expect_test _ =
  print_endline "Does it get moved?";
  print_endline "Let's hope not.";
  [%expect {| Do not move this payload |}]
;;

let%expect_test _ =
  print_endline "Does it get moved?";
  print_endline "Let's hope not.";
  [%expect_exact {| Do not move this payload |}]
;;
