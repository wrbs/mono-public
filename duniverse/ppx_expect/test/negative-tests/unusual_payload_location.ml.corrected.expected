let%expect_test _ =
  print_endline "Does it get moved?";
  print_endline "Let's hope not.";
  [%expect

    {|
    Does it get moved?
    Let's hope not.
    |}

    ]

let%expect_test _ =
  print_endline "Does it get moved?";
  print_endline "Let's hope not.";
  [%expect_exact

    {|Does it get moved?
Let's hope not.
|}

    ]
